tools/clang/test/CodeGenSPIRV/fn.export.with.entrypoint.hlsl

```diff
@@ -1,4 +1,4 @@
-// RUN: %dxc -T as_6_6 -E main -fspv-target-env=vulkan1.3 -fcgl  %s -spirv | FileCheck %s
+// RUN: %dxc -T as_6_6 -E main -fspv-target-env=universal1.5 -fcgl  %s -spirv | FileCheck %s
```

Comment:
nit: add a note in the commit message to explain why this is changed here (I suppose due to the new vk1.3 validation rules added in SPV-tools?)